string frame_id
---
sensor_msgs/CameraInfo camera_info
